build-parent/pom.xml

```diff
@@ -369,9 +369,14 @@
             <dependency>
                 <groupId>me.escoffier.certs</groupId>
                 <artifactId>certificate-generator-junit5</artifactId>
-                <version>0.5.0</version>
+                <version>0.7.1</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>me.escoffier.certs</groupId>
+                <artifactId>certificate-generator</artifactId>
+                <version>0.7.1</version>
+            </dependency>
```

Comment:
I think we should move these projects to the Quarkus or the SmallRye org. And same for the groupId.

For tests, it was mostly OK but I wouldn't be comfortable with having some CLI/runtime stuff hosted outside of our community namespaces. And this especially since they are security components.

Both for security and for the future.

I won't require this to get the PR in but we need to have a plan and a deadline for it.

Reply:
It's still the goal. I would go to SmallRye as the project has a lot of tests, and I don't want to use Quarkus CI time for this.

Reply:
Will work on something tomorrow.

Reply:
See https://github.com/quarkusio/quarkus/pull/41665

Do not resolve this conversation - once the mentioned PR is merged, there is some work to do to transition to this new dependency.